web/modules/custom/asset_release/src/Controller/AssetReleaseController.php

```diff
@@ -73,10 +99,7 @@ public function add(RdfInterface $rdf_entity) {
    *   The access result object.
    */
   public function createAssetReleaseAccess(RdfInterface $rdf_entity) {
-    $user = $this->currentUser();
-    $membership = Og::getMembership($user, $rdf_entity);
-    // @todo: Remove check for empty after ISAICP-2369 is in.
-    return (!empty($membership) && $membership->hasPermission('create asset_release rdf_entity')) ? AccessResult::allowed() : AccessResult::forbidden();
+    return $this->ogAccess->userAccessEntity('create', $this->createNewAssetRelease($rdf_entity), $this->currentUser());
```

Comment:
I see that indeed `OgAccess::userAccessGroupContentEntityOperations()` is not going through `OgAccess::userAccess()`.
@pfrenssen, do you know then if the documentation of `OgAccessInterface::userAccess()` should be updated?
>All permission checks in OG should go through this function. This way we guarantee consistent behavior, and ensure that the superuser and group administrators can perform all actions.

Related to ticket: since this is the implementation, I will have to accept it, but I would have more liked the idea of using one of these `GroupContentOperationPermission` classes to check the permissions.
Something like `someclass::getGroupContentOperatioPermission($permission, $entity_bundle)` that would return a proper `GroupContentOperationPermission` class.